Detectors/GlobalTracking/src/MatchTPCITS.cxx

```diff
@@ -44,7 +44,9 @@
 #include "DetectorsBase/GRPGeomHelper.h"
 #include "ITStracking/IOUtils.h"
 
-#include "GPUO2Interface.h" // Needed for propper settings in GPUParam.h
+// #include "GPUO2Interface.h" // Needed for propper settings in GPUParam.h
```

Comment:
Why did you remove this header? Does it create any problems? In principle, it is still needed to get the correct defines for GPUParam, etc.

Reply:
Actually, I've commented it out when was fighting with compilation (missing param.inc) and then forgot to uncomment. But it works w/o it.